test/component/stateful-form.cy.tsx

```diff
@@ -472,4 +479,441 @@ describe("StatefulForm", () => {
       cy.findAllByLabelText("stateful-form-row").should("have.length", 2);
     });
   });
+
+  context("with width", () => {
+    const DEFAULT_COUNTRY_CODES = COUNTRY_CODES.find(
+      (data) => data.id === "US" || COUNTRY_CODES[206]
+    );
+
+    if (!DEFAULT_COUNTRY_CODES) {
+      throw new Error("Default country code 'US' not found in COUNTRY_CODES.");
+    }
+
+    interface AllCaseValueProps {
+      text: string;
+      email: string;
+      number: string;
+      password: string;
+      textarea: string;
+      rating: string;
+      check: boolean;
+      chips?: {
+        searchText: string;
+        selectedOptions: BadgeProps[];
+      };
+      color: string;
+      combo: string[];
+      date: string[];
+      file_drop_box?: File[];
+      file: File | undefined;
+      image: File | undefined;
+      money: string;
+      phone: string;
+      thumb_field: boolean;
+      togglebox: boolean;
+      signature: string;
+      capsule: string;
+      country_code?: CountryCodeProps;
+    }
+
+    const FRUIT_OPTIONS: OptionsProps[] = [
+      { text: "Apple", value: "1" },
+      { text: "Banana", value: "2" },
+      { text: "Orange", value: "3" },
+      { text: "Grape", value: "4" },
+      { text: "Pineapple", value: "5" },
+      { text: "Strawberry", value: "6" },
+      { text: "Watermelon", value: "7" },
+    ];
+
+    const valueAll: AllCaseValueProps = {
+      text: "",
+      email: "",
+      number: "",
+      password: "",
+      textarea: "",
+      rating: "",
+      check: false,
+      chips: {
+        searchText: "",
+        selectedOptions: [],
+      },
+      color: "",
+      combo: [""],
+      date: [""],
+      file_drop_box: [] as File[],
+      file: undefined,
+      image: undefined,
+      money: "",
+      phone: "",
+      thumb_field: false,
+      togglebox: false,
+      signature: "",
+      capsule: "",
+      country_code: DEFAULT_COUNTRY_CODES,
+    };
+
+    const badgeSchema = z.object({
+      id: z.string().optional(),
+      metadata: z.record(z.unknown()).optional(),
+      variant: z.string().optional(),
+      withCircle: z.boolean().optional(),
+      caption: z.string().optional(),
+      backgroundColor: z.string().optional(),
+      textColor: z.string().optional(),
+      circleColor: z.string().optional(),
+    });
+
+    const schema = z.object({
```

Comment:
just wondering do we need schema here? if it's of no use for the test, let's remove it :) also the `badgeSchema` let's not write code that's not really be the essential part of the test 👍